app/Domain/Integrations/Models/IntegrationModel.php

```diff
@@ -137,7 +137,10 @@ static function (IntegrationModel $integrationModel) {
 
     public function delete(): ?bool
     {
-        $this->update(['status' => IntegrationStatus::Deleted]);
+        // Temporarily disable events to avoid triggering 'updated'
+        static::withoutEvents(function () {
+            $this->update(['status' => IntegrationStatus::Deleted]);
+        });
```

Comment:
`$this->updateQuietly(`
Could be even cleaner
https://laravel.com/docs/11.x/eloquent#saving-a-single-model-without-events